app/javascript/components/casino.vue

```diff
@@ -39,12 +53,33 @@ export default {
   data() {
     return {
       monkeys: null,
+      lastWinnerId: null,
+      thereIsWinner: false,
+      showWinner: false,
     };
   },
   mounted() {
     getMonkeys(this.casinoId).then((response) => (this.monkeys = response));
   },
   methods: {
+    bet() {
+      postBet(this.casinoId).then((response) => {
+        this.thereIsWinner = true;
+        this.showWinner = false;
```

Comment:
Me da la impresión de que basta con solo tener el `showWinner`, ya creo que el `thereIsWinner` es equivalente a decir `!this.showWinner && this.lasWinnerId`